### PR TITLE
Move to our new licensing regime.

### PR DESCRIPTION
**Pull request by ltratt**
All copyright in this project is held by King's, so moving it is trivial.

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+of their contributions is retained by an organisation:
+
+    Edd Barrett <vext01@gmail.com>: copyright retained by
+      King's College London
+    Jacob Hughes <jh@jakehughes.uk>: copyright retained by
+      King's College London
+    Sarah Mount <sarah.mount@kcl.ac.uk>: copyright retained by
+      King's College London
+    Laurence Tratt <laurie@tratt.net>: copyright retained by
+      King's College London
```
Comment (ptersilie) [inline COPYRIGHT]: I'm missing again. And I don't think Edd or Jake have any contributions in this repo, unless you want to put them there for future proofing.

Comment (ltratt) [reply COPYRIGHT]: Oops, you're quite right! Fixing...

Comment (ltratt) [reply COPYRIGHT]: Fixed in https://github.com/softdevteam/grmtools/pull/118/commits/360f87e816f5ff4acf7194bc9ce70aabdfa58037

**Comment by ptersilie** [comment]
Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**